site/content/docs/4.6/components/forms.md

```diff
@@ -19,7 +19,7 @@ Here's a quick example to demonstrate Boosted's form styles. Keep reading for do
   <div class="form-group">
     <label for="exampleInputEmail1">Email address</label>
     <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter email">
-    <small id="emailHelp" class="form-text text-muted">We'll never share your email with anyone else.</small>
+    <div id="emailHelp" class="form-text">We'll never share your email with anyone else.</div>
```

Comment:
I took the example of v5 (which come from Bootstrap). It comes from https://github.com/twbs/bootstrap/pull/30598. Since I agree with you, I'll try to make a PR on Bootstrap directly for v5.

Reply:
It is a `<small>` in Bootstrap v4. It is a `<div>` in Bootstrap v5/Boosted v5. IMO, since Bootstrap chose not to backport this modification, you should keep the same HTML element (a `<small>`) to avoid inconsistency between migration guides; or not backporting at all this feature.

Reply:
Last commit revert those changes but keep the removal of `.text-muted` for accessibility concerns.